### ISSUE TITLE
Unknown or unsupported linux distribution: rhel (v0.6.8)

### ISSUE THREAD

**Issue by opoplawski** [FrostyX/tracer#57]
This is on a Scientific Linux 7.2 box:

```
# tracer
You are running unsupported linux distribution

Please visit https://github.com/FrostyX/tracer/issues
and create new issue called 'Unknown or unsupported linux distribution: rhel (v0.6.8)' if there isn't such.
```

I tried to setup "rhel" -> yum, but now I just get:

```
PYTHONPATH=build/lib/ build/scripts-2.7/tracer
Traceback (most recent call last):
  File "build/scripts-2.7/tracer", line 34, in <module>
    tracer.main.run()
  File "/export/home/orion/fedora/tracer/tracer/build/lib/tracer/main.py", line 42, in run
    return router.dispatch()
  File "/export/home/orion/fedora/tracer/tracer/build/lib/tracer/resources/router.py", line 52, in dispatch
    controller = DefaultController(self.args, self.packages)
  File "/export/home/orion/fedora/tracer/tracer/build/lib/tracer/controllers/default.py", line 50, in __init__
    System.package_manager(erased=args.erased),
  File "/export/home/orion/fedora/tracer/tracer/build/lib/tracer/resources/system.py", line 81, in package_manager
    return PackageManager(*list(map(get_instance, managers[distro])))
  File "/export/home/orion/fedora/tracer/tracer/build/lib/tracer/resources/system.py", line 62, in get_instance
    return getattr(module, name)(**kwargs)
AttributeError: 'module' object has no attribute 'Yum'
```

**Comment by FrostyX** [comment]
Hi @opoplawski,
thank you for trying to port Tracer to Scientific Linux.

I guess that you modified [system.py:package_manager](https://github.com/FrostyX/tracer/blob/master/tracer/resources/system.py#L55)
But it is also necessary to modify the condition on top of files [rpm.py](https://github.com/FrostyX/tracer/blob/master/tracer/packageManagers/rpm.py) and [yum.py](https://github.com/FrostyX/tracer/blob/master/tracer/packageManagers/yum.py)

I am gonna have to implement a better way of doing this.

**Comment by sean797** [comment]
This was fixed in `tracer-0.6.12`. If your issue hasn't been resolved feel free to reopen it. Thanks @opoplawski 👍
